src/graphics/color.rs

```diff
@@ -31,10 +31,42 @@ impl Color {
         a: 1.0,
     };
 
+    /// Red color.
+    pub const RED: Self = Self {
+        r: 1.0,
+        g: 0.0,
+        b: 0.0,
+        a: 1.0,
+    };
+
+    /// Green color.
+    pub const GREEN: Self = Self {
+        r: 0.0,
+        g: 1.0,
+        b: 0.0,
+        a: 1.0,
+    };
+
+    /// Blue color.
+    pub const BLUE: Self = Self {
+        r: 0.0,
+        g: 0.0,
+        b: 1.0,
+        a: 1.0,
+    };
+
     /// Creates a new [`Color`] from components in the [0, 1.0] range.
     ///
     /// [`Color`]: struct.Color.html
     pub fn new(r: f32, g: f32, b: f32, a: f32) -> Color {
+        debug_assert!(r >= 0.0, "Color values should be in the [0, 1.0] range");
+        debug_assert!(r <= 1.0, "Color values should be in the [0, 1.0] range");
+        debug_assert!(g >= 0.0, "Color values should be in the [0, 1.0] range");
+        debug_assert!(g <= 1.0, "Color values should be in the [0, 1.0] range");
+        debug_assert!(b >= 0.0, "Color values should be in the [0, 1.0] range");
+        debug_assert!(b <= 1.0, "Color values should be in the [0, 1.0] range");
+        debug_assert!(a >= 0.0, "Color values should be in the [0, 1.0] range");
+        debug_assert!(a <= 1.0, "Color values should be in the [0, 1.0] range");
```

Comment:
I would probably customize the panic message for each component, like: "R value was < 0.0".